buildSrc/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ repositories {
 	mavenCentral()
 	gradlePluginPortal()
 	maven(url = "https://repo.gradle.org/gradle/enterprise-libs-release-candidates-local/")
+	maven(url = "https://jitpack.io")
```

Comment:
Not needed due to:
https://github.com/junit-team/junit5/blob/eaab286a53d534966de7cf6200955c0d3b8830f4/build.gradle.kts#L116-L120